docs/source/issues.rst

```diff
@@ -55,7 +59,18 @@ ElasticBLAST uses cloud storage to temporally store query sequences and internal
    gsutil -m rm gs://${ELB_RESULTS_BUCKET}/logs/*  # to delete metadata files
 
 
+.. _early_shutdown:
+
 A synchronous search may shut down too early
 --------------------------------------------
 
 When doing a synchronous search (submitted with the ``--sync`` option), ElasticBLAST is continuously probing for search status to know when the search is done. When the status check times out ElasticBLAST interprets it as search failure and shuts down the cluster.
+
+.. _too_many_jobs:
+
+Too many jobs leads to a failed execution
+-----------------------------------------
+
+If the ElasticBLAST configuration leads to too many (e.g.: over 5,000)
+kubernetes jobs created, ElasticBLAST will disconnect from the cluster and it
```

Comment:
Is it our code that disconnects from the cluster?

Reply:
I haven't investigated this, but the fact is that there is a disconnect.

Reply:
That was my worry, if there is a disconnect we may be not deleting the cluster.

Reply:
that is not the case in the case I observed, but this needs to be investigated further